### ISSUE TITLE
Implement ReLU for meta Tensor

### ISSUE THREAD

**Issue by xiurobert** [pytorch/pytorch#79938]
### 🚀 The feature, motivation and pitch

It looks like `aten::relu` is not currently implemented for meta tensors.

Running the code in #79848 is almost possible if using branch with #79834, however it looks like ReLU is not supported on meta. This seems a bit strange, since `Sigmoid` seems to be supported.

### Alternatives

Remove activation when using meta device.

### Additional context

```python

import torch

feature_extractor = torch.nn.Sequential(
    torch.nn.Conv2d(3, 8, kernel_size=3),
    torch.nn.ReLU(),
    torch.nn.MaxPool2d(kernel_size=2),
    torch.nn.Conv2d(8, 16, kernel_size=4),
    torch.nn.ReLU(),
    torch.nn.MaxPool2d(kernel_size=3)
)

b_img = torch.rand(32, 3, 28, 28, device='meta')
feature_extractor.to('meta')

print(feature_extractor(b_img).shape)
```

is not possible while

```python

import torch

feature_extractor = torch.nn.Sequential(
    torch.nn.Conv2d(3, 8, kernel_size=3),
    torch.nn.Sigmoid(),
    torch.nn.MaxPool2d(kernel_size=2),
    torch.nn.Conv2d(8, 16, kernel_size=4),
    torch.nn.Sigmoid(),
    torch.nn.MaxPool2d(kernel_size=3)
)

b_img = torch.rand(32, 3, 28, 28, device='meta')
feature_extractor.to('meta')

print(feature_extractor(b_img).shape)
```
works fine

cc @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are

**Comment by xiurobert** [comment]
Nice. I'll try to get a PR in tomorrow.

**Comment by ngimel** [comment]
We have a decomposition for relu in primTorch, so there should be possible to run meta shape propagation for it, but unfortunately I don't know the exact syntax off the top of my head.

**Comment by ezyang** [comment]
According to our test suite relu is supposed to work on master but I am able to reproduce the error on the script

**Comment by ezyang** [comment]
Oh it's dumb, the problem is we don't import all the refs modules on `import torch`
